### ISSUE TITLE
Screen reader does not read tooltip content on field.

### ISSUE THREAD

**Issue by swirtSJW** [department-of-veterans-affairs/va.gov-cms#7146]
## Describe the defect
The tooltip field formatter introduced on https://github.com/department-of-veterans-affairs/va.gov-cms/issues/6449
does not meet accessibility standards because the tooltip content is not available to screen readers.

The tooltip implementation used on our fieldset does get read appropriately by a screen reader

https://github.com/department-of-veterans-affairs/va.gov-cms/issues/6449

This is not caught by AXE tests, but was caught by an actual screen reader.

## Expected behavior
Screen reader will have be able to read tooltip content.

## Screenshots 
![image](https://user-images.githubusercontent.com/5752113/144546965-cdb21737-5a9e-48a4-9e97-561900bc6308.png)

 ## Additional context
Add any other context about the problem here. Reach out to the Product Managers to determine if it should be escalated as critical (prevents users from accomplishing their work with no known workaround and needs to be addressed within 2 business days).

 
## Labels
(You can delete this section once it's complete)
- [x] Issue type (red) (defaults to "Defect")
- [ ] CMS subsystem (green)
- [ ] CMS practice area (blue)
- [x] CMS workstream (orange) (not needed for bug tickets)
- [ ] CMS-supported product (black)

### CMS Team

Please leave only the team that will do this work selected. If you're not sure, it's fine to leave both selected.

- [ ] `Platform CMS Team`
- [x] `Sitewide CMS Team`
